playwright-e2e/dss/pages/page-base.ts

```diff
@@ -44,7 +45,7 @@ export default abstract class PageBase implements PageInterface {
   }
 
   async waitForReady(): Promise<void> {
-    await this.page.waitForLoadState('networkidle');
```

Comment:
Removing `networkidle` because usage is discouraged from Playwright documentation because it's flaky.